binary-search-tree/node-js/inventory-management.js

```diff
@@ -10,6 +10,10 @@ class Product {
     constructor({id, inventoryCount}) {
         this.id = id;
         this.inventoryCount = inventoryCount;
+        this.inventoryCount = 0;
+    }
+    inventoryCount() {
+        this.inventoryCount ++; 
     }
```

Comment:
The exercise spec doesn't ask you to do this. Please reflect on why you added this, and how it might be contributing to the failing test

Reply:
I think this can be a lesson
1. In situational awareness. If you look around the codebase and observe how `update_inventory` is being called you'll notice that it takes two arguments, product ID and new inventory count.
2. In asking for clarification from the spec

Reply:
yes because i knew that it takes `inventoryCount` as a param, therefore in my head it made sense that a return value for it